framework/include/utils/RankFourTensorImplementation.h

```diff
@@ -455,6 +456,51 @@ RankFourTensorTempl<T>::transposeMajor() const
   return result;
 }
 
+template <typename T>
+RankFourTensorTempl<T>
+RankFourTensorTempl<T>::transposeIJ() const
```

Comment:
Should this be `transposeIj` with our naming conventions for these methods?

Reply:
Thanks for the review @bwspenc . You are right, I updated the name as suggested